README.md

```diff
@@ -1,264 +1,40 @@
-### 4.2.0 ###
-
-* Added: Unit tests for the `ActivityPub\Transformer\Post` class
-* Improved: Reuse constants once they're defined
-* Improved: "FEP-b2b8: Long-form Text" support
-* Improved: Admin notice for plain permalink settings is more user-friendly and actionable
-* Improved: Post-Formats support
-* Fixed: Do not display ActivityPub's user sub-menus to users who do not have the capabilities of writing posts.
-* Fixed: Proper margins for notices and font size for page title in settings screen.
-* Fixed: Ensure that `?author=0` resolves to blog user
-
-### 4.1.1 ###
-
-* Fixed: Only revert to URL if there is one
-* Fixed: Migration
-
-### 4.1.0 ###
-
-* Added: Add custom Preview for "Fediverse"
-* Added: Support `comment_previously_approved` setting
-* Fixed: Hide sticky posts that are not public
-* Improved: `activity_handle_undo` action
-* Improved: Add title to content if post is a `Note`
-* Improved: Fallback to blog-user if user is disabled
-
-### 4.0.2 ###
-
-* Fixed: Do not federate "Local" posts
-* Improved: Help-text for Content-Warning box
-
-### 4.0.1 ###
-
-* Fixed: Missing URL-Param handling in REST API
-* Fixed: Seriously Simple Podcasting integration
-* Fixed: Multiple small fixes
-* Improved: Provide contextual fallback for dynamic blocks
-
-### 4.0.0 ###
-
-* Added: Fire an action before a follower is removed
-* Added: Make Intent-URL filterable
-* Added: `title` attribute to link headers for better readability
-* Added: Post "visibility" feature
-* Added: Attribution-Domains support
-* Improved: Inbox validation
-* Improved: WordPress-Post-Type - Detection
-* Improved: Only validate POST params and do not fall back to GET params
-* Improved: ID handling for a better compatibility with caching plugins
-* Fixed: The "Shared Inbox" endpoint
-* Fixed: Ensure that sticky_posts is an array
-* Fixed: URLs and Hashtags in profiles were not converted
-* Fixed: A lot of small improvements and fixes
-
-See full Changelog on [GitHub](https://github.com/Automattic/wordpress-activitypub/blob/trunk/CHANGELOG.md).
-
-## Upgrade Notice ##
+> [WordPress Playground](https://wordpress.org/playground/) is the platform that lets you run WordPress instantly on any device without a host. It’s your place to build, experiment, test, and grow.
 
-### 1.0.0 ###
+## Documentation
 
-For version 1.0.0 we have completely rebuilt the followers lists. There is a migration from the old format to the new, but it may take some time until the migration is complete. No data will be lost in the process, please give the migration some time.
+WIP.
 
-## Installation ##
+## Federation
 
-Follow the normal instructions for [installing WordPress plugins](https://wordpress.org/support/article/managing-plugins/).
+ActivityPub is a protocol for federated social networks, enabling communication between different platforms. For details on what the plugin supports, refer to the [FEDERATION.md](.FEDERATION.md) file.
 
-### Automatic Plugin Installation ###
+## Contribute
```

Comment:
I wonder if folks would benefit from a "Support" section before this one, inviting folks to post in the WordPress.org support forums if they need any help?